docs/release_notes/release_notes-0.16.0.rst

```diff
@@ -52,6 +52,7 @@ Key Features and Updates
 * New Features
   * FEAT-4463: Add experimental fuzzydata integration for testing against a randomized dataframe workflow (#4556)
   * FEAT-#4419: Extend virtual partitioning API to pandas on Dask (#4420)
+  * FEAT-#4670:  Implement convert_dtypes by mapping across partitions (#4671)
```

Comment:
```suggestion
  * FEAT-#4670: Implement convert_dtypes by mapping across partitions (#4671)
```

Reply:
This release note should probably go under `Pandas API implementations and improvements`.